resources/es/cemu.po

```diff
@@ -2151,19 +2490,19 @@ msgstr "&Ir al volcado"
 
 #: SaveImportWindow.cpp
 msgid "Import save entry"
-msgstr "Importe la entrada de la partida guardada"
+msgstr "Importar entrada de partida guardada"
 
 #: SaveImportWindow.cpp SaveTransfer.cpp
 msgid "Source"
 msgstr "Origen"
 
 #: SaveImportWindow.cpp
 msgid "Select a zipped save file"
-msgstr "Seleccione un archivo de guardado comprimido"
+msgstr "Seleccione un archivo de partida guardada comprimida"
 
 #: SaveImportWindow.cpp
 msgid "Import save entry {}"
-msgstr "Importar entrada de la partida guardada {}"
+msgstr "Entrada de partida guardada para importar {}"
```

Comment:
Could you tell me where this string is used in the application and why did you change it like that? Because I translated it like you did before, but honestly I'm not sure what is being put in place of `{}`.

Reply:
When importing a file to compare the checksum

Reply:
Ok, thanks!